### PR TITLE
add link to groups listed in TSC/working_groups.md

### PR DESCRIPTION
**Pull request by pedrovsp**
I added to the getting started page the working groups listed in TSC/working_groups.md and other working groups that I was able to identify.

### CODE REVIEW

File: contribute_to_node.md [hunk 1]
```diff
@@ -9,8 +9,22 @@ and we'll label your request as `intro requested`. A collaborator will facilitat
 ### Group contributing guides
 - [Website WG](https://github.com/nodejs/nodejs.org/blob/master/CONTRIBUTING.md#nodejs-community-contributing-guide-10)<br>
   The Website Working Group is primarily concerned with the code and overall structure of the [Node.js](https://nodejs.org) website.
+- [Streams WG](https://github.com/nodejs/readable-stream/blob/master/README.md#streams-working-group)<br>
+  The Streams Working Group is dedicated to the support and improvement of the Streams API as used in Node.js and the npm ecosystem.
+- [Build WG](https://github.com/nodejs/build#nodejs-build-working-group)<br>
+  The Node.js Build Working Group maintains and controls infrastructure used for continuous integration (CI), releases, benchmarks, web hosting (of nodejs.org and other Node.js web properties) and more.
+- [Diagnostics WG](https://github.com/nodejs/diagnostics#diagnostics-working-group)<br>
```
Comment (vsemozhetbyt) [inline contribute_to_node.md]: It seems this group is already mentioned below)

File: contribute_to_node.md [hunk 2]
```diff
@@ -9,8 +9,22 @@ and we'll label your request as `intro requested`. A collaborator will facilitat
...
+  The goal of this WG is to ensure Node provides a set of comprehensive, documented, extensible diagnostic protocols, formats, and APIs to enable tool vendors to provide reliable diagnostic tools for Node.
+- [i18n](https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md#i18n)<br>
+  The i18n Working Groups handle more than just translations. Each working group is an endpoint for community members to collaborate with each other in their language of choice.
+- [Docker WG](https://github.com/nodejs/docker-node#nodejs)<br>
+  The Docker Working Group's purpose is to build, maintain, and improve official Docker images for the Node.js project.
+- [Adddon API WG](https://github.com/nodejs/nan#native-abstractions-for-nodejs)<br>
```
Comment (vsemozhetbyt) [inline contribute_to_node.md]: A typo: Adddon -> Addon )

**Review by vsemozhetbyt — APPROVED** [review]
Thank you! LGTM with 2 nits)

**Review by keywordnew — COMMENTED** [review]
@pedrovspereira Thank you for taking the time and effort to contribute here!

The PR looks good to me. It should be ready to merge when you resolve the comments made above.

**Review by bnb — COMMENTED** [review]
LGTM once the requested changes are made! Thanks for your work @pedrovspereira 💥

**Comment by pedrovsp** [comment]
Thanks for the review! Just resolved the issues.

**Comment by keywordnew** [comment]
Merged. ⚡️ Thanks for your efforts @pedrovspereira!

**Pull request merged**